rules/integrations/aws/exfiltration_s3_bucket_policy_added_for_external_account_access.toml

```diff
@@ -0,0 +1,66 @@
+[metadata]
+creation_date = "2024/04/17"
+integration = ["aws"]
+maturity = "production"
+min_stack_comments = "AWS integration breaking changes, bumping version to ^2.0.0"
+min_stack_version = "8.9.0"
+updated_date = "2024/04/17"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies an AWS S3 bucket policy change to share permissions with an external account. Adversaries may attempt to
+backdoor an S3 bucket by sharing it with an external account. This can be used to exfiltrate data or to provide access
+to other adversaries. This rule identifies changes to a bucket policy via the `PutBucketPolicy` API call where the
+policy includes an `Effect=Allow` statement that does not contain the AWS account ID of the bucket owner.
+"""
+false_positives = [
+    """
+    Legitimate changes to share an S3 bucket with an external account may be identified as false positive but are not
+    best practice.
+    """,
+]
+index = ["filebeat-*", "logs-aws.cloudtrail-*"]
+language = "eql"
+license = "Elastic License v2"
+name = "AWS S3 Bucket Policy Added to Share with External Account"
+references = [
+    "https://stratus-red-team.cloud/attack-techniques/AWS/aws.exfiltration.s3-backdoor-bucket-policy/",
+    "https://docs.aws.amazon.com/AmazonS3/latest/API/API_PutBucketPolicy.html",
+]
+risk_score = 47
+rule_id = "e8c9ff14-fd1e-11ee-a0df-f661ea17fbce"
+setup = "S3 data event types must be collected in the AWS CloudTrail logs."
+severity = "medium"
+tags = [
+    "Domain: Cloud",
+    "Data Source: AWS",
+    "Data Source: Amazon Web Services",
+    "Data Source: AWS S3",
+    "Use Case: Threat Detection",
+    "Tactic: Exfiltration",
+]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+any where event.dataset == "aws.cloudtrail" and event.provider == "s3.amazonaws.com"
```

Comment:
We may need to create an issue request to integrations to add categories based on the `event.provider` or `event.type`. For example, `iam` or `sts` can be considered `authentication`. A bigger question would be how do we categorize cloud events in general compared to endpoint?